docs/deployment/setup.md

```diff
@@ -0,0 +1,118 @@
+# KV-Cache Manager Setup Guide
+
+This guide provides a complete walkthrough for setting up and testing the example llm-d-kv-cache-manager system. You will deploy a vLLM with LMCache and Redis, then run an example application that demonstrates KV cache indexing capabilities.
+
+By following this guide, you will:
+
+1. **Deploy the Infrastructure**: Use Helm to set up:
+   - vLLM servers (default: 4 replicas) serving Llama 3.1 8B Instruct model
```

Comment:
```suggestion
   - vLLM nodes with LMCache CPU offloading (4 replicas) serving Llama 3.1 8B Instruct model
```

Apologies for missing earlier, LMCache being a standalone item makes it sound like it will run separately. The LMCache engine is initialized within the vLLM processes.

Reply:
Thank you for pointing out this! 🙏 i have updated the code